FRAuth/FRAuth/User/Browser.swift

```diff
@@ -392,16 +392,16 @@ import SafariServices
     }
 
     
-    /// Performs authentication through /authorize endpoint using SFAuthenticationsession
+    /// Performs authentication through /authorize endpoint using ASWebAuthenticationSession
     /// - Parameters:
     ///   - url: URL of /authorize including all URL query parameter
     ///   - completion: Completion callback to nofiy the result
     /// - Returns: Boolean indicator whether or not launching external user-agent was successful
     func loginWithSFWebSession(url: URL, completion: @escaping UserCallback) -> Bool {
```

Comment:
Can we not fully delete this `loginWithSFWebSession` method? In the current SDK I think that this is only used for iOS 11

Reply:
It does not seem to used in the file

Reply:
@george-bafaloukas-forgerock you are right. it's no longer used. will remove